pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -533,6 +533,14 @@ func (cm *containerManagerImpl) Start(node *v1.Node,
 	// allocatable of the node
 	cm.nodeInfo = node
 
+	rootfs, err := cm.cadvisorInterface.RootFsInfo()
+	if err != nil {
+		return fmt.Errorf("failed to get rootfs info: %v", err)
+	}
+	for rName, rCap := range cadvisor.EphemeralStorageCapacityFromFsInfo(rootfs) {
```

Comment:
shouldn't this be gated by the feature gate?

Reply:
The feature gate which prevents adding Local Storage to node Capacity and Allocatable is in `kubelet_node_status.go`.  This PR doesn't change that.  Although I agree that having the capacity in the container manager be different from the capacity on the node status is confusing and bug-prone.  I can add another feature gate here if you think that is a good idea.

Reply:
@dashpole the problem we are tracking is a regression in podstartup time - https://github.com/kubernetes/kubernetes/issues/60589#issuecomment-374236943 looks like this PR could be contributing to it right?

Reply:
@dims this may contribute to node startup, but pod startup would seem quite odd.

Reply:
But the scalability tests use a mock cadvisor, so this shouldn't contribute to node startup latency in those tests either.

Reply:
Ack. thanks @dashpole will keep looking